image_recognition_openpose/src/image_recognition_openpose/openpose_wrapper.py

```diff
@@ -45,7 +45,10 @@ def __init__(self, model_folder, pose_model, net_input_size, net_output_size, nu
 
         logging.info("Loading openpose with parameters: %s", parameters)
 
-        self._openpose = openpose.OpenPose(parameters)
+        self._op = op
```

Comment:
I think this is not needed, as it isn't used.

Reply:
Used in line 62 from what I see in the diff here

Reply:
I think it can be deleted here and line 61 can be changed to `datum = op.Datum()`. Can you test it and push the commit if it works?